tests/testFakeProcessing.py

```diff
@@ -145,12 +148,15 @@ def testFakeProcessing(self):
 
             exposure = pCcdResult.resultList[0].result.calibRes.exposure
             mask = exposure.getMaskedImage().getMask()
-            maskBit = 2**mask.getMaskPlaneDict()['FAKE']
+            maskBit = 2**mask.getMaskPlaneDict()[maskPlaneName]
```

Comment:
I can't get that to work so I think I'm going to punt on that fix for the moment.